api/server/router/container/copy.go

```diff
@@ -16,6 +16,10 @@ import (
 
 // postContainersCopy is deprecated in favor of getContainersArchive.
 func (s *containerRouter) postContainersCopy(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
+	version := httputils.VersionFromContext(ctx)
+	if version.GreaterThanOrEqualTo("1.24") {
+		return fmt.Errorf("/containers/(id or name)/copy is not supported for this API version : %s", version)
```

Comment:
Not sure this is the right error — as discussed a little bit with @thaJeztah, a simple HTTP 404 error should be fine.